libs/langchain/langchain/retrievers/multi_query.py

```diff
@@ -22,18 +22,18 @@ class LineListOutputParser(BaseOutputParser[List[str]]):
 
     def parse(self, text: str) -> List[str]:
         lines = text.strip().split("\n")
-        return lines
+        return list(filter(str.strip, lines))
 
 
 # Default prompt
 DEFAULT_QUERY_PROMPT = PromptTemplate(
     input_variables=["question"],
-    template="""You are an AI language model assistant. Your task is 
-    to generate 3 different versions of the given user 
-    question to retrieve relevant documents from a vector  database. 
-    By generating multiple perspectives on the user question, 
-    your goal is to help the user overcome some of the limitations 
-    of distance-based similarity search. Provide these alternative 
+    template="""You are an AI language model assistant. Your task is
```

Comment:
To fix the prompt to not add extra whitespace, either use `"` for concatenation (instead of `"""`)
or else start the text without the leading whitespace

```python
TEXT = """"\
My text\n
Some more text
""""
```

Or

```python
TEXT = (
  "My text\n"
  "More text\n"
)
```